api/v2alpha1/pipeline_spec.proto

```diff
@@ -5,6 +5,7 @@ package ml_pipelines;
 import "google/protobuf/duration.proto";
 import "google/protobuf/struct.proto";
 import "google/rpc/status.proto";
+import "common.proto";

```

Comment:
cc @chensun may I get your thoughts on this 

Initially I was copying the proto files, but I've switch to this approach. 

This is to reduce duplication of the api by having both packages share the common spec.

This requires us to use the local packages for testing [here](https://github.com/kubeflow/pipelines/pull/11621/files#diff-72cd9e9641f7a4163c9e79ea5f170f176ac00dad62af3c0cca8d8c9ee13c3787R32-R37) (which requires protobuf-compiler in the via [this](https://github.com/kubeflow/pipelines/pull/11621/files#diff-5487a622aae74a1f9c34bb344fd9b216e441caa710e1381d4d9855dc533804afR32)). 

Let me know your thoughts and if there are concerns around this approach!

Reply:
Nevermind I reverted it to copying the proto for now, while I got it working with shared protos, the change is not trivial and probably requires further discussion, in the future I will raise this as a separate PR (it maybe moot if in the future we plan to combine these packages anyways).